index.mdx

```diff
@@ -2,129 +2,4 @@
-  - [Other nodes](/docs/explain/other-nodes)
-      * [Aggregate](/docs/explain/other-nodes/aggregate)
-      * [Append](/docs/explain/other-nodes/append)
-      * [Bitmap And](/docs/explain/other-nodes/bitmap-and)
-      * [Bitmap Or](/docs/explain/other-nodes/bitmap-or)
-      * [Gather Merge](/docs/explain/other-nodes/gather-merge)
-      * [Gather](/docs/explain/other-nodes/gather)
-      * [Group](/docs/explain/other-nodes/group)
-      * [Hash](/docs/explain/other-nodes/hash)
-      * [Limit](/docs/explain/other-nodes/limit)
-      * [Lock Rows](/docs/explain/other-nodes/lock-rows)
-      * [Materialize](/docs/explain/other-nodes/materialize)
-      * [Merge Append](/docs/explain/other-nodes/merge-append)
-      * [Modify Table](/docs/explain/other-nodes/modify-table)
-      * [Project Set](/docs/explain/other-nodes/project-set)
-      * [Recursive Union](/docs/explain/other-nodes/recursive-union)
-      * [Result](/docs/explain/other-nodes/result)
-      * [SetOp](/docs/explain/other-nodes/set-op)
-      * [Sort](/docs/explain/other-nodes/sort)
-      * [Unique](/docs/explain/other-nodes/unique)
-      * [Window Aggregate](/docs/explain/other-nodes/window-aggregate)
-
-* **[pganalyze Enterprise Server](/docs/enterprise)**
-  - Setup instructions
-    - [Initial setup on a VM](/docs/enterprise/setup)
-    - [Initial setup on Kubernetes (Amazon EKS)](/docs/enterprise/setup/kubernetes_amazon_eks)
-    - [Object storage setup](/docs/enterprise/setup/object-storage)
-    - [Separate collector installation](/docs/enterprise/setup/separate-collector-install)
-    - [Google Auth setup](/docs/enterprise/google-auth)
-    - [PagerDuty and Slack integration setup](/docs/enterprise/integrations)
-    - [Container settings](/docs/enterprise/settings)
-  - [Release changelog](/docs/enterprise/releases)
-  - [Upgrading to new releases](/docs/enterprise/upgrade)
-  - [Reference architecture](/docs/enterprise/architecture)
-  - [Troubleshooting](/docs/enterprise/troubleshooting)
-
-* **[pganalyze GraphQL API](/docs/api)**
-  - [Creating an API key](/docs/api/create-api-key)
-  - [Queries](/docs/api/queries)
-      * [getIssues - Get check-up issues and alerts](/docs/api/queries/getIssues)
-      * [getQueryStats - Export query statistics](/docs/api/queries/getQueryStats)
-  - [Mutations](/docs/api/mutations)
-      * [addServer - Add a server to pganalyze Enterprise Server](/docs/api/mutations/addServer)
-
-<>{/* **[Guides](/docs/guides)**
-  - [Tuning checkpoint intervals to reduce I/O spikes](/docs/guides/tuning-checkpoint-intervals)
-  - [Adjusting work_mem based on temporary file creation](/docs/guides/adjusting-work-mem)
-  - [Exporting query statistics using the pganalyze API](/docs/guides/exporting-query-statistics)
-  - [Monitoring Postgres locks using Log Insights](/docs/guides/monitoring-postgres-locks-using-log-insights)
-*/}</>
-
----
-
-* [Open-Source Components](/docs/open_source_components)
+Please select a navigation item on the left.
```

Comment:
Note this is intended to be merged at the same time as the website change that introduces the side navigation as a replacement.